qemu/tests/block_hotplug.py

```diff
@@ -130,7 +132,23 @@ def get_new_disk(disk1, disk2):
 
             error_context.context("Check block device after hotplug.",
                                   logging.info)
-            if params.get("pci_test_cmd"):
+            if params.get("os_type") == "windows":
+                iozone_cmd = params.get("iozone_cmd") % (disk_letter[num],
+                                                         disk_letter[num])
+                session = vm.wait_for_login(timeout=timeout)
+                if i == 0:
+                    if params.get("format_disk", "no") == "yes":
+                        error_context.context("Format disk", logging.info)
+                        utils_misc.format_windows_disk(session,
+                                                       disk_index[num],
+                                                       mountpoint=disk_letter[num])
+                error_context.context("Start iozone in guest.", logging.info)
+                s, o = session.cmd_status_output(iozone_cmd,
+                                                 timeout=(iozone_timeout))
+                session.close()
+                if s:
+                    test.fail("Failed to run iozone, output: %s" % o)
+            else:
                 if params.get("os_type") == "linux":
                     test_cmd = params.get("pci_test_cmd") % (disk, disk)
```

Comment:
You have judge os_type is windows in line135,  so in line151, it should be linux already, no need to judge if it is linux guest in line152. It will lead line154 always false.